style.css

```diff
@@ -437,4 +437,382 @@ input[type=email] {
   .fontawesome {
     display: none;
   }
+
+  .uber-navig {
+    display: flex;
+    flex-direction: column;
+    width: 345px;
+    height: 596px;
+    padding: 16px;
+    margin: 25px 24px;
+    border-radius: 16px;
+    border: 1px solid #dfe1e6;
+    box-sizing: border-box;
+    background-color: #fff;
+    justify-content: center;
+  }
+}
+
+@media screen and (min-width: 1000px) {
+  body {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+    width: 1600px;
+  }
+
+  .header {
+    height: 280px;
+    background-color: #fff;
+    padding: 25px 155px;
+  }
+
+  #nav-bar {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin: 0 auto;
+    padding-top: 20px;
+    align-items: center;
+  }
+
+  .nav-buttons {
+    display: flex;
+    list-style-type: none;
+    cursor: pointer;
+  }
+
+  .nav-buttons a {
+    text-decoration: none;
+    height: 2em;
+    padding: 0.5em 1em;
+  }
+
+  .nav-buttons a:hover {
+    text-decoration: none;
+    height: 2em;
+    color: rgb(62, 151, 96);
+  }
+
+  main {
+    position: relative;
+    background-color: #e5e5e5;
+  }
+
+  #main-page {
+    height: 100%;
+    background-size: cover;
+    background-repeat: no-repeat;
+    width: 100%;
+    margin: 10px auto;
+    padding-bottom: 30rem;
+    padding-top: 1rem;
+  }
```

Comment:
- [ ]  id is used for single elements that appear on the page for only once. Kindly, let us move our ids to classes.